packages/react-components/react-table/src/components/DataGridBody/useDataGridBody.tsx

```diff
@@ -17,6 +17,7 @@ export const useDataGridBody_unstable = (props: DataGridBodyProps, ref: React.Re
   const { sortable } = useTableContext();
   const getRows = useDataGridContext_unstable(ctx => ctx.getRows);
   const sort = useDataGridContext_unstable(ctx => ctx.sort.sort);
+  // ColumnId context not valid here.
```

Comment:
would you mind expanding a little on this? Is the columnId used in the sort(getrows()) portion?